submissions/Roophee/Popup_checkbox/index.html

```diff
@@ -0,0 +1,484 @@
+                                    <li class="popup__item">
+                                        <a href="#">
+                                            <svg display="none" xmlns="http://www.w3.org/2000/svg">
+
+                                                <symbol id="document" viewBox="0 0 512  512 ">
+                                                    <path
+                                                        d="M428 224H288a48 48 0 0 1-48-48V36a4 4 0 0 0-4-4h-92a64 64 0 0 0-64 64v320a64 64 0 0 0 64 64h224a64 64 0 0 0 64-64V228a4 4 0 0 0-4-4zm-92 160H176a16 16 0 0 1 0-32h160a16 16 0 0 1 0 32zm0-80H176a16 16 0 0 1 0-32h160a16 16 0 0 1 0 32z" />
+                                                    <path
+                                                        d="M419.22 188.59L275.41 44.78a2 2 0 0 0-3.41 1.41V176a16 16 0 0 0 16 16h129.81a2 2 0 0 0 1.41-3.41z" />
+
+                                                </symbol>
+
+                                            </svg>
+                                            <svg class="popup__item-icon" xmlns="http://www.w3.org/2000/svg">
+                                                <use xlink:href="#document">
+                                                </use>
+                                            </svg>
+                                        </a>
+                                        <span>Document</span>
+                                    </li>
+                                    <li class="popup__item-more-button nav__button--focus">
+                                        <label for="popup__checkbox2" class="popup__checkbox2-label"><span
+                                                class="hidden">More button</span>
+                                            <svg display="none" xmlns="http://www.w3.org/2000/svg">
+
+                                                <symbol id="more1" viewBox="0 0 24 24 ">
+                                                    <g>
+                                                        <circle cx="12" cy="12" r="2" />
+                                                        <circle cx="19" cy="12" r="2" />
+                                                        <circle cx="5" cy="12" r="2" />
+                                                    </g>
+
+                                                </symbol>
+
+                                            </svg>
+                                            <svg class="popup__item-icon popup__item-icon--more-button "
+                                                xmlns="http://www.w3.org/2000/svg">
+                                                <use xlink:href="#more1">
+                                                </use>
+                                            </svg>
+
+                                        </label>
+                                        <input type="checkbox" name="popup__checkbox2" class="popup__checkbox2-check"
```

Comment:
Watch the [video](https://www.youtube.com/watch?v=E6kLaaQFctU&t=413s) about hiding checkboxes in the right way.

Reply:
That was my problem. I didn`t understand how to arrange elements and selectors. But I did it. Thanks.

Reply:
[Read more](https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-within) about how `:focus-within` works.